packages/amazonq/test/unit/codewhisperer/commands/onAcceptance.test.ts

```diff
@@ -20,11 +20,13 @@ import { assertTelemetryCurried } from 'aws-core-vscode/test'
 describe('onAcceptance', function () {
     describe('onAcceptance', function () {
         beforeEach(async function () {
+            const session = CodeWhispererSessionState.instance.getSession()
```

Comment:
```suggestion
        const session = CodeWhispererSessionState.instance.getSession()

        describe('onAcceptance', function () {
        beforeEach(async function () {
```

Does this need to be duplicated? Applicable throughout PR.

Reply:
yes that's a good callout, thanks Max